lib/profiler.ts

```diff
@@ -0,0 +1,22 @@
+import fetch from 'node-fetch'
+
+const BACKEND_URL = 'https://superhuman.com/~backend'
```

Comment:
By baking in more superhman specifics - are we moving away from zen being a general purpose testing library?
Could/should we add this as a hook instead, allowing the application to control this instead of the testing library? 

Do we want this going to production?

Reply:
Right now that is the setup I had for testing locally but I like your idea. Right now the zen.config does pass in as JS not json so we could add a `log` field that would do the metric stuff